lib/send_text.rb

```diff
@@ -0,0 +1,33 @@
+require 'twilio-ruby'
+
+class SendText
+
+  attr_reader :customers
+
+  def confirmation_text
+    sender
+  end
+
+  private
+
+    def sender
+
+      accountSID = 'AC8a18c4b3cf182c308e952100a2a8c1ab'
+      authToken = '5e8d102aef932d5a67bcc4e786e0e084'
```

Comment:
Style/VariableName: Use snake_case for variable names. (https://github.com/bbatsov/ruby-style-guide#snake-case-symbols-methods-vars)